.circleci/BUILD.bazel

```diff
@@ -0,0 +1,50 @@
+load("@aspect_bazel_lib//lib:write_source_files.bzl", "write_source_file")
+
+CIRCLECI_ORG = "aspect-build"
+
+CIRCLECI_USER_CONFIG_FILE = "//.circleci:user-config.yml"
+
+alias(
+    name = "rosetta",
+    actual = select({
+        "@bazel_tools//src/conditions:darwin_arm64": "@rosetta_darwin_aarch64//file:rosetta",
+        "@bazel_tools//src/conditions:darwin_x86_64": "@rosetta_darwin_x86_64//file:rosetta",
+        "@bazel_tools//src/conditions:linux_aarch64": "@rosetta_linux_aarch64//file:rosetta",
+        "@bazel_tools//src/conditions:linux_x86_64": "@rosetta_linux_x86_64//file:rosetta",
+    }),
+)
+
+alias(
```

Comment:
Ah, I studied https://github.com/theoremlp/rules_multitool at one point and I think this may be a good opportunity to use it. Reduces all this boilerplate nicely without us needing a ruleset

Reply:
There is still the sets of hashes in deps.bzl and the genrule in the BUILD file that really belong in a rule set so the DX is cleaner for Workflows deployments. I'd rather dep on a aspect_rules_workflows or aspect_rules_cicd and remove all of the complexity than dep on a rules_multitool and remove only a portion of it.

Reply:
I'm also thinking this pattern also makes sense to use for GitHub Actions to reduce the complexity we have right now with the setup step generating json that feeds into the re-usable workflow matrix. Stamping out the generated GHA workflows would be much much simpler. That also seems better than a generic GitHub Actions that runs the setup step using rosetta which generates json and feeds into the re-usable workflow.

Reply:
I'd still like to push on the "intended" way to use GHA. This is one of those cases where we spent a lot of time evaluating all the options for the way we are doing it now. We can escalate to someone about the inability to use a reusable workflow from another GH org.

Reply:
Also I'm still not sold on us publishing another ruleset, that's signing us up for more overhead and maintenance, we have too many already for the amount of automation we have. I'd want to understand all the other things that will go in there, not just this one thing.
I know, by objecting I'm volunteering to propose something that works just as well...